### PR TITLE
Update placement-and-order.mdx

### PR DESCRIPTION
**Pull request by lucasveil**

### CODE REVIEW

File: apps/storefront/docs/patterns/placement-and-order/placement-and-order.mdx [hunk 1]
```diff
@@ -4,3 +4,27 @@ metaTitle: Placement and order
 mode: draft
 route: /patterns/placement-and-order/
 ---
+## Guidelines
+In order to place yout objects and component the right way, you have always to keep in mind the ready pattern of your user. 2 types of patterns can be encountered: F and Z.
```
Comment (BeckyBrekke) [inline apps/storefront/docs/patterns/placement-and-order/placement-and-order.mdx:8]: A few mistakes here, re-write it to say: 
In order to place your objects and components correctly, you must always keep in mind the reading pattern of your user. Two types of patterns can be encountered: F and Z.

File: apps/storefront/docs/patterns/placement-and-order/placement-and-order.mdx [hunk 2]
```diff
@@ -4,3 +4,27 @@ metaTitle: Placement and order
...
+
+### The F reading pattern
+
+In the F-shaped scanning pattern is characterized by many fixations concentrated at the top and the left side of the page.
+
+Users first read in a horizontal movement, usually across the upper part of the content area. He will then
+move down the page a bit and then read across in a second horizontal movement that typically covers a shorter area than the previous movement. This additional element forms the F’s lower bar.
```
Comment (BeckyBrekke) [inline apps/storefront/docs/patterns/placement-and-order/placement-and-order.mdx:15]: A few British spellings, and updates to keep it consistent with the Z reading pattern paragraph: 

>~~In t~~The F-shaped scanning pattern is characteri~~z~~sed by many fixations concentrated at the top and the left side of the page.
> 
> Users first read in a horizontal movement, usually across the upper part of the content area. ~~He~~They will then
> move down the page a bit and then read across in a second horizontal movement that typically covers a shorter area than the previous movement. This additional element forms the F’s lower bar.

File: apps/storefront/docs/patterns/placement-and-order/placement-and-order.mdx [hunk 3]
```diff
@@ -4,3 +4,27 @@ metaTitle: Placement and order
...
+
+The main interaction should then be aligned on the left.
+
+### The Z reading pattren
+
+The z-pattern layout follows the shape of the letter Z. A z-pattern design traces the route the human eye travels when they scan the page— left to right, top to bottom.
+
+Users first read in a horizontal movement, usually across the upper part of the content area then 
+down and to the left side of the page, creating a diagonal line and back across to the right again, forming a second horizontal line.
```
Comment (BeckyBrekke) [inline apps/storefront/docs/patterns/placement-and-order/placement-and-order.mdx:24]: Some changes to make it consistent with the previous paragraph: 

> The ~~z~~Z-shaped scanning pattern ~~layout follows the shape of the letter Z. A z-pattern design~~ traces the route the human eye travels when they scan the page — left to right, top to bottom.

Users first read in a horizontal movement, usually across the upper part of the content area. They will then move
down and to the left side of the page~~,~~ creating a diagonal line, and then move back across to the right again~~,~~ forming a second horizontal line.

**Review by BeckyBrekke — CHANGES_REQUESTED** [review]
I few things to change. I re-wrote the text a little to make it consistent,

File: apps/storefront/docs/patterns/placement-and-order/placement-and-order.mdx (hunk 1 above)
Comment (lucasveil) [reply apps/storefront/docs/patterns/placement-and-order/placement-and-order.mdx:8]: Updated

File: apps/storefront/docs/patterns/placement-and-order/placement-and-order.mdx (hunk 2 above)
Comment (lucasveil) [reply apps/storefront/docs/patterns/placement-and-order/placement-and-order.mdx:15]: Updated

File: apps/storefront/docs/patterns/placement-and-order/placement-and-order.mdx (hunk 3 above)
Comment (lucasveil) [reply apps/storefront/docs/patterns/placement-and-order/placement-and-order.mdx:24]: Updated

**Pull request merged**